### PR TITLE
feat: or-787 fixed elasticsearch tests

### PR DESCRIPTION
**Pull request by W0dan**

### CODE REVIEW

File: test/OrganisationRegistry.ElasticSearch.Tests/EnvVarIgnoreFactAttribute.cs [hunk 1]
```diff
@@ -7,10 +7,10 @@ public class EnvVarIgnoreFactAttribute : FactAttribute
     {
         public EnvVarIgnoreFactAttribute()
         {
-            string envVar = "ES_TESTS";
+            const string envVar = "IGNORE_ES_TESTS";
             var env = Environment.GetEnvironmentVariable(envVar);
-            if (string.IsNullOrEmpty(env))
-                Skip = $"Please set {envVar} env var to run.";
+            if (!string.IsNullOrEmpty(env) && env.Trim().ToUpper() != "FALSE")
```
Comment (koenmetsu) [inline test/OrganisationRegistry.ElasticSearch.Tests/EnvVarIgnoreFactAttribute.cs:12]: Why not `string.Equals("", "", StringComparison.InvariantCultureIgnoreCase)` ?

File: test/OrganisationRegistry.ElasticSearch.Tests/EnvVarIgnoreFactAttribute.cs [hunk 2]
```diff
@@ -7,10 +7,10 @@ public class EnvVarIgnoreFactAttribute : FactAttribute
     {
         public EnvVarIgnoreFactAttribute()
         {
-            string envVar = "ES_TESTS";
+            const string envVar = "IGNORE_ES_TESTS";
             var env = Environment.GetEnvironmentVariable(envVar);
```
Comment (koenmetsu) [inline test/OrganisationRegistry.ElasticSearch.Tests/EnvVarIgnoreFactAttribute.cs:11]: Not your code, but the naming here is weird. `ignoreEsTests` maybe?

File: test/OrganisationRegistry.ElasticSearch.Tests/OrganisationRegistry.ElasticSearch.Tests.csproj [hunk 3]
```diff
@@ -3,6 +3,9 @@
   <Import Project="..\..\packages\Be.Vlaanderen.Basisregisters.Build.Pipeline\Content\Be.Vlaanderen.Basisregisters.Build.Pipeline.Settings.Test.props" />
   <ItemGroup>
     <None Update="appsettings.json" CopyToOutputDirectory="Always" />
+    <None Update="appsettings.Wodan-Ranger.json">
```
Comment (koenmetsu) [inline test/OrganisationRegistry.ElasticSearch.Tests/OrganisationRegistry.ElasticSearch.Tests.csproj:6]: <Content Include="appsettings.*.json" CopyToOutputDirectory="Always" />

File: test/OrganisationRegistry.ElasticSearch.Tests/Scenario/OrganisationScenario.cs [hunk 4]
```diff
@@ -1,6 +1,8 @@
 namespace OrganisationRegistry.ElasticSearch.Tests.Scenario
 {
     using System;
+    using System.Collections.Generic;
```
Comment (koenmetsu) [inline test/OrganisationRegistry.ElasticSearch.Tests/Scenario/OrganisationScenario.cs:4]: Are these necessary?

**Review by koenmetsu — COMMENTED** [review]
Minor changes requested

File: test/OrganisationRegistry.ElasticSearch.Tests/EnvVarIgnoreFactAttribute.cs (hunk 2 above)
Comment (W0dan) [reply test/OrganisationRegistry.ElasticSearch.Tests/EnvVarIgnoreFactAttribute.cs:11]: om consistent te zijn met de andere env vars ?

**Pull request merged**